Buildings/Controls/OBC/ASHRAE/PrimarySystem/ChillerPlant/Tower/FanSpeed/EnabledWSE/Subsequences/WSEOperation.mo

```diff
@@ -0,0 +1,287 @@
+  parameter Modelica.SIunits.Time Ti=0.5 "Time constant of integrator block"
+    annotation (Dialog(group="Chilled water controller",
+                       enable=chiWatCon==Buildings.Controls.OBC.CDL.Types.SimpleController.PI or
+                              chiWatCon==Buildings.Controls.OBC.CDL.Types.SimpleController.PID));
+  parameter Modelica.SIunits.Time Td=0.1 "Time constant of derivative block"
+    annotation (Dialog(group="Chilled water controller",
+                       enable=chiWatCon==Buildings.Controls.OBC.CDL.Types.SimpleController.PD or
+                              chiWatCon==Buildings.Controls.OBC.CDL.Types.SimpleController.PID));
+
+  Buildings.Controls.OBC.CDL.Interfaces.RealInput uTowSpe(
+    final min=0,
+    final max=1,
+    final unit="1") "Tower fan speed"
+    annotation (Placement(transformation(extent={{-180,80},{-140,120}}),
+      iconTransformation(extent={{-140,60},{-100,100}})));
+  Buildings.Controls.OBC.CDL.Interfaces.RealInput TChiWatSup(
+    final unit="K",
+    final quantity="ThermodynamicTemperature")
+    "Chilled water supply temperature"
+    annotation (Placement(transformation(extent={{-180,-60},{-140,-20}}),
+      iconTransformation(extent={{-140,-20},{-100,20}})));
+  Buildings.Controls.OBC.CDL.Interfaces.RealInput TChiWatSupSet(
+    final unit="K",
+    final quantity="ThermodynamicTemperature")
+    "Chilled water supply temperature setpoint"
+    annotation (Placement(transformation(extent={{-180,-140},{-140,-100}}),
+      iconTransformation(extent={{-140,-100},{-100,-60}})));
+  Buildings.Controls.OBC.CDL.Interfaces.RealOutput yTowSpe(
+    final min=0,
+    final max=1,
+    final unit="1")
+    "Tower fan speed when WSE is enabled and there is any chiller running"
+    annotation (Placement(transformation(extent={{140,40},{180,80}}),
+      iconTransformation(extent={{100,-20},{140,20}})));
+
+protected
+  Buildings.Controls.OBC.CDL.Logical.And and2 "Logical and"
+    annotation (Placement(transformation(extent={{40,130},{60,150}})));
+  Buildings.Controls.OBC.CDL.Logical.Timer tim
+    "Count the time when fan is at minimum speed and the chilled water supply temperature drops below setpoint"
+    annotation (Placement(transformation(extent={{80,130},{100,150}})));
+  Buildings.Controls.OBC.CDL.Continuous.GreaterEqualThreshold greEquThr(
+    final threshold=300)
+    "Check if fan is at minimum speed and the chilled water supply temperature drops below setpoint for 5 minutes"
+    annotation (Placement(transformation(extent={{80,90},{100,110}})));
+  Buildings.Controls.OBC.CDL.Continuous.Sources.Constant minTowSpe(
+    final k=minSpe) "Minimum tower speed"
+    annotation (Placement(transformation(extent={{-120,10},{-100,30}})));
+  Buildings.Controls.OBC.CDL.Continuous.Feedback feedback1 "Input difference"
+    annotation (Placement(transformation(extent={{-90,90},{-70,110}})));
+  Buildings.Controls.OBC.CDL.Continuous.Hysteresis hys2(
```

Comment:
For the hold block, do you mean `TrueFalseHold`? It will be not sufficient here. The output is able to hold at lease 3 minutes, but cannot correctly switch back to true.
 
The condition (`CHWST rises above setpoint by 1 F`) for switching the output (fan status) back from `false` (cycle off) to `true` (turn on) is not the condition used for cycle off fan (`CHWST drop below setpoint and fans have been at minimum speed for 5 minutes`).

@milicag 
In fact, regarding the text `If CHWST drops below setpoint and fans have been at minimum fan speed for 5 minutes`, do you think it is 
`If (CHWST drops below setpoint and fans have been at minimum fan speed) for 5 minutes`, or
`If (CHWST drops below setpoint) and (fans have been at minimum fan speed for 5 minutes)`?